### DISCUSSION TITLE
tseslint/consistent-type-imports doesn't work with eslint@9 #9578

### DISCUSSION THREAD

**Discussion by vaynevayne** [typescript-eslint/typescript-eslint#9579] [RFCs]
### RFC

Suggested changes...

### Additional Info

Any additional info...

### Before you submit your RFC, please confirm the following. If any of these required steps are not taken, we may not be able to review your RFC. Help us to help you!

- [X] I have [read the discussions guidelines](https://typescript-eslint.io/contributing/discussions) and per those guidelines, this fits a discussion category, _not_ a help request or standard issue.
- [X] I have [searched for related discussions](https://github.com/typescript-eslint/typescript-eslint/discussions) and [searched for related issues](https://github.com/typescript-eslint/typescript-eslint/issues) and found none that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/linting/troubleshooting) and my problem is not listed.

**Comment by bradzacher** [comment]
Please don't spam us by filing duplicate discussions. #9578
